Library/Sender.ts

```diff
@@ -312,6 +321,11 @@ class Sender {
 
                     if (typeof callback === "function") {
                         if (error) {
+                            // If the error type is a timeout we want to provide more meaningful output
```

Comment:
What is original error name and message here?

Reply:
The original is: 
```
{ message: 'socket hang up', name: 'Error'" }
```